rm-services.yml

```diff
@@ -258,7 +258,7 @@ services:
      - LIQUIBASE_URL=jdbc:postgresql://postgres:${POSTGRES_PORT}/${POSTGRES_HOST}
      - JAVA_OPTS=-Xmx128m -Xdebug -Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=${NOTIFY_GATEWAY_DEBUG_PORT}
     healthcheck:
-      test: ["CMD", "curl", "-f", "http://localhost:8181/info"]
+      test: ["CMD", "curl", "-f", "-XGET", "http://localhost:8181/info"]
```

Comment:
should there be a space between the flag and the HTTP verb?

Reply:
Possibly I can see it stays as unhealthy. Will take another look today

Reply:
It looks good, just needed to add curl to the survey image https://github.com/ONSdigital/rm-survey-service/pull/26